Zend/zend.h

```diff
@@ -351,6 +351,16 @@ ZEND_API void zend_save_error_handling(zend_error_handling *current);
 ZEND_API void zend_replace_error_handling(zend_error_handling_t error_handling, zend_class_entry *exception_class, zend_error_handling *current);
 ZEND_API void zend_restore_error_handling(zend_error_handling *saved);
 
+typedef void (*zend_error_notify_cb)(int type, const char *error_filename, uint32_t error_lineno, zend_string *message);
+
+BEGIN_EXTERN_C()
+
+void zend_register_error_notify_callback(zend_error_notify_cb callback);
```

Comment:
Should it be ZEND_API?

Reply:
Yeah, it should.

Reply:
oh damn, will fix it

Reply:
Ah you already committed this yesterday, thanks